docs/installation.md

```diff
@@ -0,0 +1,95 @@
+> ⚠️ This project is very alpha stage. Expect bugs and incomplete documentation.
+
+The following instructions are intended for contributors who want to try Metals
+on a real-world project.
```

Comment:
"...project. We do not currently provide support for day-to-day usage of Metals."?

Reply:
Done.

Reply:
NB. these instructions are copy-pasted from the old BETA.md, they will need to be mostly rewritten soon